en/docs/setup/wso2is_script.sql

```diff
@@ -0,0 +1,83 @@
+
+ALTER TABLE `apimgt`.`AM_SUBSCRIBER` CHANGE COLUMN `CREATED_TIME` `CREATED_TIME` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ,CHANGE COLUMN `UPDATED_TIME` `UPDATED_TIME` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ;
+
+ALTER TABLE `apimgt`.`AM_APPLICATION` CHANGE COLUMN `UPDATED_TIME` `UPDATED_TIME` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ,CHANGE COLUMN `UUID` `UUID` VARCHAR(256) NULL DEFAULT NULL ;
+
+ALTER TABLE `apimgt`.`AM_API` CHANGE COLUMN `UPDATED_TIME` `UPDATED_TIME` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ;
+
+ALTER TABLE `apimgt`.`AM_SUBSCRIPTION` CHANGE COLUMN `CREATED_TIME` `CREATED_TIME` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP , CHANGE COLUMN `UPDATED_TIME` `UPDATED_TIME` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP , CHANGE COLUMN `UUID` `UUID` VARCHAR(256) NULL DEFAULT NULL ;
+
+ALTER TABLE `apimgt`.`AM_SUBSCRIPTION_KEY_MAPPING` CHANGE COLUMN `KEY_TYPE` `KEY_TYPE` VARCHAR(512) NOT NULL ;
+
+ALTER TABLE `apimgt`.`AM_APPLICATION_KEY_MAPPING` CHANGE COLUMN `KEY_TYPE` `KEY_TYPE` VARCHAR(512) NOT NULL ;
+
+ALTER TABLE `apimgt`.`AM_WORKFLOWS` CHANGE COLUMN `WF_CREATED_TIME` `WF_CREATED_TIME` TIMESTAMP NULL ;
+
+ALTER TABLE apimgt.AM_ALERT_TYPES_VALUES DROP INDEX AM_ALERT_TYPES_VALUES_CONSTRAINT;
+
+ALTER TABLE apimgt.AM_ALERT_TYPES_VALUES ADD PRIMARY KEY (ALERT_TYPE_ID,USER_NAME,STAKE_HOLDER);
+
+ALTER TABLE apimgt.AM_ALERT_EMAILLIST DROP INDEX AM_ALERT_EMAILLIST_CONSTRAINT;
+
+ALTER TABLE `apimgt`.`AM_ALERT_EMAILLIST` DROP PRIMARY KEY, ADD PRIMARY KEY (`EMAIL_LIST_ID`, `USER_NAME`, `STAKE_HOLDER`);
+
+ALTER TABLE apimgt.AM_ALERT_EMAILLIST_DETAILS DROP INDEX AM_ALERT_EMAILLIST_DETAILS_CONSTRAINT;
+
+ALTER TABLE apimgt.AM_ALERT_EMAILLIST_DETAILS ADD PRIMARY KEY (EMAIL_LIST_ID,EMAIL);
+
+
+ALTER TABLE `apimgt`.`AM_POLICY_SUBSCRIPTION` CHANGE COLUMN `NAME` `NAME` VARCHAR(512) NOT NULL , CHANGE COLUMN `UUID` `UUID` VARCHAR(256) NULL DEFAULT NULL ;
+
+ALTER TABLE `apimgt`.`AM_POLICY_APPLICATION` CHANGE COLUMN `NAME` `NAME` VARCHAR(512) NOT NULL , CHANGE COLUMN `UUID` `UUID` VARCHAR(256) NULL DEFAULT NULL ;
+
+ALTER TABLE `apimgt`.`AM_POLICY_HARD_THROTTLING` CHANGE COLUMN `NAME` `NAME` VARCHAR(512) NOT NULL ;
+
+ALTER TABLE `apimgt`.`AM_API_THROTTLE_POLICY` CHANGE COLUMN `NAME` `NAME` VARCHAR(512) NOT NULL , CHANGE COLUMN `UUID` `UUID` VARCHAR(256) NULL DEFAULT NULL ;
+
+ALTER TABLE `apimgt`.`AM_POLICY_GLOBAL` CHANGE COLUMN `NAME` `NAME` VARCHAR(512) NOT NULL , CHANGE COLUMN `UUID` `UUID` VARCHAR(256) NULL DEFAULT NULL ;
+
+
+ALTER TABLE `apimgt`.`AM_BLOCK_CONDITIONS` CHANGE COLUMN `UUID` `UUID` VARCHAR(256) NULL DEFAULT NULL ;
+
+create index IDX_AAI_CTX on apimgt.AM_API (CONTEXT);
+
+create index IDX_AAUM_TT on apimgt.AM_API_URL_MAPPING (THROTTLING_TIER);
+
+
+ALTER TABLE `regdb`.`REG_LOG` DROP INDEX `REG_LOG_IND_BY_REG_LOGTIME` , ADD INDEX `REG_LOG_IND_BY_REG_LOG` USING HASH (`REG_LOGGED_TIME`, `REG_TENANT_ID`);

```

Comment:
I think we should divide this into multiple scripts based on the DB. So each can execute in the respective DB. And in the instructions section, we need to inform that each should execute in the correct DB.